parsers/arc_plc_parser.py

```diff
@@ -0,0 +1,201 @@
+                                })
+
+                            # only include this commodity if at least one program exists
+                            if program_entries:
+                                commodity_records[commodity_name] = {
+                                    "commodityName": commodity_name,
+                                    "baseAcres": round(county_total_base_acres, 2),
+                                    "programs": program_entries
+                                }
+
+                        # only include this scenario if at least one commodity exists
+                        if commodity_records:
+                            total_payments_for_county = sum(
+                                prog["totalPaymentInDollars"]
+                                for com in commodity_records.values()
+                                for prog in com["programs"]
+                            )
+                            scenario_records[scenario_name] = {
+                                "scenarioName": scenario_name,
+                                "totalPaymentInDollars": round(total_payments_for_county, 2),
+                                "commodities": list(commodity_records.values())
+                            }
+
+                    # only include this county if at least one scenario exists
+                    if scenario_records:
+                        state_county_records.append({
+                            "countyFIPS": county_fips,
+                            "scenarios": list(scenario_records.values())
+                        })
+
+                # only include this state if at least one county exists
+                if state_county_records:
+                    total_state_payments = sum(
+                        scen["totalPaymentInDollars"]
+                        for c in state_county_records
+                        for scen in c["scenarios"]
+                    )
+                    output_by_year[model_year].append({
+                        "state": state_abbr,
+                        "totalPaymentInDollars": round(total_state_payments, 2),
+                        "counties": state_county_records
+                    })
+
+        # Write gzipped JSON
+        json_bytes = json.dumps(output_by_year, indent=2, sort_keys=True).encode("utf-8")
+        output_path = os.path.join(self.data_folder, f"arc_plc_payments_{scenario_label}.json.gz")
+        with gzip.open(output_path, "wb") as gzip_file:
+            gzip_file.write(json_bytes)
+
+
+if __name__ == '__main__':
+    # Get this data from the box folder, I left it out because these are very large files
+    # TODO consider making these files input parameters so this can later be used as part of a workflow
+    # NOTE: Since these are big files, please find them in Box
+    fiscal_year = 2025
+    start_year = 2025
+    end_year = 2035
+    current_farmbill_data = "../title-1-commodities/arcplc_model/CSVResultsCurrentFB06-24-2025.csv"
```

Comment:
I don't see these files, should they be added to this PR?

Reply:
Let's discuss in slack. For now, box is fine. Just as long as they are somewhere. I couldn't find them on box either. I just wanted to make sure we had the data somewhere for others to get at.